src/components/AccessibilityBody/AccessibilityBody.test.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Enzyme from 'enzyme';
+import AccessibilityBody from './index';
+
+const mount = Enzyme.mount;
+
+let wrapper;
+
+describe('<AccessibilityBody />', () => {
+  describe('renders', () => {
+    beforeEach(() => {
+      wrapper = mount(
+        <AccessibilityBody
+          communityAccessibilityLink="http://example.com"
+          phoneNumber="555-555-5555"
+          email="example@example.com"
+        />,
+      );
+    });
+    it('contains links', () => {
+      expect(wrapper.text()).toContain('example@example.com');
+      expect(wrapper.text()).toContain('555-555-5555');
```

Comment:
Is there a way to add an expect for the communityAccessibilityLink?

Reply:
Sure thing!

I'll add in something like:
```
      expect(wrapper.find({ href: 'http://example.com' })).toHaveLength(1);
```